package.json

```diff
@@ -22,28 +23,28 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build && attw --pack",
+    "build": "tsc && vite build && attw --pack --profile esm-only",
```

Comment:
There is no CJS build, only ESM and UMD, which is not recognised by ATTW yet

https://github.com/arethetypeswrong/arethetypeswrong.github.io/issues/146

Reply:
fixed by making CJS extension - recommended way in Vite docs:
https://vite.dev/guide/build.html#library-mode

Reply:
second declaration file was required for UMD/CJS build, using this DTS plugin for that:
https://dev.to/rashidshamloo/exporting-declaration-files-in-vitejs-f6m